_pages/Beep.md

```diff
@@ -1,8 +1,19 @@
 ---
 layout: page
-title: Beep routine
+title: beep_* routines
 permalink: /beep/
 exclude: true
 ---
 
+The beep routines are the base for all [beep patterns](#beep-patterns) that are created by the system.
+
+there are normal versions of each beep and short versions of each beep. The higher the number, the higher the tone of the beep.
+
+> Eg.: beep_f4 plays a higher note than beep_f2. beep_f1 and beep_f1_short are the same note, just that the short note only plays for about half as long as the normal one.
```

Comment:
@damosvil Can you fact check me here please? F4 is a higher note than F2, right?

Reply:
Yes, F4 period is shorter than F2 one, so it is a higher note.